fdbclient/BackupAgentBase.actor.cpp

```diff
@@ -223,8 +223,9 @@ Key getApplyKey(Version version, Key backupUid) {
 	return k2.withPrefix(applyLogKeys.begin);
 }
 
-Key getLogKey(Version version, Key backupUid) {
-	int64_t vblock = (version - 1) / CLIENT_KNOBS->LOG_RANGE_BLOCK_SIZE;
+Key getLogKey(Version version, Key backupUid, int blockSize) {
```

Comment:
there is a comment here https://github.com/apple/foundationdb/blob/main/fdbclient/BackupAgentBase.actor.cpp#L162, does it also apply to this place?

Reply:
yes. I added a comment about this.